obspy/io/sac/arrayio.py

```diff
@@ -514,7 +514,8 @@ def dict_to_header_arrays(header=None, byteorder='='):
                     #                                          'strict')
                     hs[HD.STRHDRS.index(hdr)] = value
             else:
-                raise ValueError("Unrecognized header name: {}.".format(hdr))
+                msg = "Unrecognized header name: {}. Ignored.".format(hdr)
```

Comment:
I think this becomes less readable as one line, as two things are happening, and it'll go over 72 columns.  OK if it stays as is?